WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,55 @@
+
+## Daily routine
+
+1. [Check code review requests](https://github.com/zkSNACKs/WalletWasabi/pulls). __Reviews / Awaiting review from you__.
+2. Check [priority issues and PR-s!](https://github.com/zkSNACKs/WalletWasabi/labels/priority). Take it or set Assignees and Labels if you can.
+3. Check your [own PR-s and answer](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+repo%3AzkSNACKs%2FWalletWasabi+) if necessary. __Click Created__.
+   - If you need help use __@mention__.
+   - If you need review __set Reviewers__.
+4. Check [assigned issues](https://github.com/zkSNACKs/WalletWasabi/issues) __Click Assignee / Select your profile__.
+5. Check [mentions](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+repo%3AzkSNACKs%2FWalletWasabi+) __Click Mentioned__.
+6. Check the bell.
```

Comment:
I think instead of step 1 to 6, we should do this from the GitHub notification tab. Everyone should move to the new notification tab as it's much better.

![image](https://user-images.githubusercontent.com/9156103/72456939-0108c800-379c-11ea-9630-7700e1b7eb89.png)

Reply:
@yahiheb can you check if you can use the new notifications now?

Reply:
https://github.com/notifications/beta

Reply:
No I still can't use it.